.github/workflows/python-test.yml

```diff
@@ -1,6 +1,13 @@
 name: Python Tests
 
-on: [push]
+on:
```

Comment:
This would be nice so I CI can validate the changes for pull requests. I assume it needs to be merged before it takes affect. Let me know if you do not want this change

Reply:
Not sure what the direct impact of this is. Does this run within the context of our organisation?

Reply:
My intended effect is that forked PRs have CI run as well so we can make sure our changes work, and it should run with the same context as the existing workflow